polaris-service/src/main/java/io/polaris/service/auth/OAuthUtils.java

```diff
@@ -16,31 +16,11 @@
 package io.polaris.service.auth;
 
 import jakarta.ws.rs.core.Response;
-import java.nio.charset.StandardCharsets;
-import org.apache.commons.codec.binary.Base64;
 
 /** Simple utility class to assist with OAuth operations */
 public class OAuthUtils {
-
-  public static final String AUTHORIZATION_HEADER = "Authorization";
-
-  public static final String SF_HEADER_ACCOUNT_NAME = "Snowflake-Account";
```

Comment:
This looks like a Snowflake specific string that arguably doesn't need to live here any more, but first I'd like to verify nothing we have internally depends on it before removing it. Let me check on that.

Reply:
Confirmed, we still use these internally. Let me try to quickly remove our internal dependency on these and then we can move forward here.

Reply:
Sorry for the delay. This is all cleaned up on our side now.